contracts/datafeeds/DataFeedInterface.sol

```diff
@@ -10,6 +10,8 @@ contract DataFeedInterface {
 
     // Get registartion specific information
     function isDataSet(address ofAsset) constant returns (bool) {}
+    function isDataValid(address ofAsset) constant returns (bool) {}
+    function isValid(address ofAsset) constant returns (bool) {}
```

Comment:
@travs `isValid` should be listed under comment `// Get asset specific information` not `// Get registartion specific information`

Reply:
done, and noticed that `// Get asset specific information` was specified twice for no obvious reason, so merged under one comment. I can change that back if there was some reason I don't understand